multiqc/utils/search_patterns.yaml

```
@@ -257,6 +257,8 @@ fastqc/theoretical_gc:
 featurecounts:
     fn: '*.summary'
     shared: true
+fgbio/groupreadsbyumi:
+    fn: '*.histo.tsv'
```

Instead of searching by filename, I think we can look for the presence of the string `fraction_gt_or_eq_family_size` in the first line of the file. This is a pretty specific column title, so should be unique to this tool.

This makes it easier for people to run with the tool output, and also makes it less likely for the module to trip up with other histogram data.

```suggestion
    contents: 'fraction_gt_or_eq_family_size'
    num_lines: 1
```

---

Please also remove the other help text about needing the filename pattern if you agree with the above 👍